app/src/main/java/com/kms/cura/utils/PatinetProfileUtils.java

```diff
@@ -0,0 +1,33 @@
+package com.kms.cura.utils;
+
+import com.kms.cura.entity.user.PatientUserEntity;
+
+import java.sql.Date;
+
+/**
+ * Created by linhtnvo on 6/8/2016.
+ */
+public class PatinetProfileUtils {
```

Comment:
Also, if this class is to contain the current user then change it to something appropriately